app/web/features/messages/MessagesHeader.tsx

```diff
@@ -0,0 +1,85 @@
+import { TabContext } from "@material-ui/lab";
+import HtmlMeta from "components/HtmlMeta";
+import NotificationBadge from "components/NotificationBadge";
+import PageTitle from "components/PageTitle";
+import TabBar from "components/TabBar";
+import MarkAllReadButton from "features/messages/requests/MarkAllReadButton";
+import { useRouter } from "next/router";
+import { ReactNode } from "react";
+import { messagesRoute, MessageType } from "routes";
+import makeStyles from "utils/makeStyles";
+
+import useNotifications from "../useNotifications";
+import { MESSAGES } from "./constants";
+
+const useStyles = makeStyles((theme) => ({
+  tabBarContainer: {
+    display: "flex",
+    justifyContent: "flex-start",
+  },
+}));
+
+export function MessagesNotification() {
+  const { data } = useNotifications();
+
+  return (
+    <NotificationBadge count={data?.unseenMessageCount}>
+      Chats
+    </NotificationBadge>
+  );
+}
+
+export function HostRequestsReceivedNotification() {
+  const { data } = useNotifications();
+
+  return (
+    <NotificationBadge count={data?.unseenReceivedHostRequestCount}>
+      Hosting
+    </NotificationBadge>
+  );
+}
+
+export function HostRequestsSentNotification() {
+  const { data } = useNotifications();
+
+  return (
+    <NotificationBadge count={data?.unseenSentHostRequestCount}>
+      Surfing
+    </NotificationBadge>
+  );
+}
+
+const labels: Record<MessageType, ReactNode> = {
+  //all: "All",
+  chats: <MessagesNotification />,
+  hosting: <HostRequestsReceivedNotification />,
+  surfing: <HostRequestsSentNotification />,
+  //meet: "Meet",
+  //archived: "Archived",
+};
+
+export default function MessagesHeader({
+  tab,
```

Comment:
```suggestion
  tab = "chats",
```
So it defaults to the first tab when the given route is `/messages` like before (also similar to what you've done for the communities page tabs)